internal/controller/config/config.go

```diff
@@ -358,3 +361,21 @@ func (config *ProgressiveConfig) GetChildStatusAssessmentSchedule(kind string) (
 	}
 	return AssessmentSchedule{}, fmt.Errorf("No Assessment Schedule found for kind %q", kind)
 }
+
+// GetAnalysisRunTimeout parses the GlobalConfig for the analysisRunTimeout string to convert into a time.Duration
+func (config *ProgressiveConfig) GetAnalysisRunTimeout() (time.Duration, error) {
+
+	// default timeout is 20 minutes
+	analysisRunTimeout := 1200
+
+	if config.AnalysisRunTimeout != "" {
+		analysisRunTimeout, err := strconv.Atoi(config.AnalysisRunTimeout)
+		if err != nil {
+			return 0, fmt.Errorf("invalid analysisRunTimeout value: %w", err)
```

Comment:
returning an error would effectively put the Controller in a crash loop. Instead let's log an Error and then just use the default